### ISSUE TITLE
Welcome Back!!  Requesting support for EthashB3 algo for Hypra

### ISSUE THREAD

**Issue by d57heinz** [todxx/teamredminer#760]
Welcome Back Team Red!!.. We look forward to your return and magic wrt AMD cards.. Hoping you can add support for EthashB3 for Hypra.  It would be great to be able to use the Bc 250s for that algo since your miner works best on anything AMD.

**Comment by VDreaming** [comment]
when support EthashB3（Hypra），Looking forward to it ！

**Comment by VDreaming** [comment]
ALPH also

**Comment by CamJava007** [comment]
Yes, definitely looking forward to ALPH being added. Right now I'm stuck using BZ if I want to Dual Mine Zil also.
